fg/settings/logging.py

```diff
@@ -14,6 +14,11 @@
 ENABLE_SENTRY=False
 SENTRY_DSN="https://xxxxxxxxxxxxxxxxxxxxxxx@sentry.io/xxxxxxx"
 
+try:
+    from .secrets import *
+except ImportError:
+    pass
+
 # Default Django logging is WARNINGS+ to console
```

Comment:
Why did you add this here? Secrets should be import from config.py (at the end) and the main entrypoint is of course the __init__.py. This should be removed.

Reply:
In your documentation you say to copy config.py and create a new file called secrets.py.

The problem with this file is that SENTRY_DSN is set right before later code that uses the variable. Thus it will override any previously set value.